activerecord/test/cases/schema_loading_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'thread'
```

Comment:
Thank you @jeremy - very good eye. I found a solution that no longer needed the thread but it got left in.